Gemfile.lock

```diff
@@ -7,41 +7,73 @@ PATH
 GEM
   remote: https://rubygems.org/
   specs:
-    diff-lcs (1.1.3)
-    ffi (1.1.5)
-    guard (1.4.0)
-      listen (>= 0.4.2)
-      thor (>= 0.14.6)
-    guard-rspec (2.1.0)
-      guard (>= 1.1)
-      rspec (~> 2.11)
-    listen (0.5.3)
-    mock_redis (0.5.4)
-    rake (0.9.2.2)
-    rb-fsevent (0.9.2)
-    rb-inotify (0.8.8)
-      ffi (>= 0.5.0)
-    redis (3.2.0)
-    rspec (2.11.0)
-      rspec-core (~> 2.11.0)
-      rspec-expectations (~> 2.11.0)
-      rspec-mocks (~> 2.11.0)
-    rspec-core (2.11.1)
-    rspec-expectations (2.11.3)
-      diff-lcs (~> 1.1.3)
-    rspec-mocks (2.11.3)
-    thor (0.16.0)
-    timecop (0.5.2)
+    coderay (1.1.3)
+    diff-lcs (1.5.0)
+    ffi (1.15.5)
+    formatador (1.1.0)
+    guard (2.18.0)
+      formatador (>= 0.2.4)
+      listen (>= 2.7, < 4.0)
+      lumberjack (>= 1.0.12, < 2.0)
+      nenv (~> 0.1)
+      notiffany (~> 0.0)
+      pry (>= 0.13.0)
+      shellany (~> 0.0)
+      thor (>= 0.18.1)
+    guard-compat (1.2.1)
+    guard-rspec (4.7.3)
```

Comment:
### [guard-rspec](https://github.com/guard/guard-rspec)

**Major** version upgrade :chart_with_upwards_trend::exclamation: 2.1.0 → 4.7.3

[_~~change-log~~, [source-code](http://github.com/guard/guard-rspec), [gem-diff](https://my.diffend.io/gems/guard-rspec/2.1.0/4.7.3)_]





<details>
<summary>Commits</summary>

A change of **403** commits. See the full changes on [the compare page](https://github.com/guard/guard-rspec/compare/v2.1.0...v4.7.3).

These are the first 10 commits:
- (f48c1c5) [Warn about deprecated :keep_failed and :focus_on_failed opti…](https://github.com/guard/guard-rspec/commit/f48c1c5c86cf0d9d9875f530ff9f2931bfb76902)
- (96739fc) [Fix :focus_on_failed deprecation message](https://github.com/guard/guard-rspec/commit/96739fc215e1df23f53e811af0f4b7d29aa0195d)
- (6d3489c) [Add .ruby-version/-gemset to .gitignore](https://github.com/guard/guard-rspec/commit/6d3489c229bdb04688919dae50ee53c1f589150e)
- (e092109) [Keep only spec paths in KeepingInspector (omit line numbers)](https://github.com/guard/guard-rspec/commit/e092109d39104dfa5ea4485e2cab54fda1bc6676)
- (c27ac21) [Merge pull request #217 from 907th/respect_guard_notifier](https://github.com/guard/guard-rspec/commit/c27ac21b956485e77e792f7dfdb40d90329460cf)
- (7d4de6d) [Update rbx on Travis](https://github.com/guard/guard-rspec/commit/7d4de6d491b4be141328c4be46da4815bb049a94)
- (3aca6f4) [Remove specific rbx version on Travis](https://github.com/guard/guard-rspec/commit/3aca6f45136d68eb03eb368488dded726b890c74)
- (01b688f) [Allow failures on jruby and rbx on Travis](https://github.com/guard/guard-rspec/commit/01b688ff49a1ba55453c3153b08f4d78d6b6b988)
- (18ceee4) [Release 4.1.0](https://github.com/guard/guard-rspec/commit/18ceee44f9bae004266b1be538f9f740b514a89c)
- (f3dab54) [Notify @thibaudgg only of Travis CI results](https://github.com/guard/guard-rspec/commit/f3dab541f2cee8c72b55c5fc450839f3596c1f9c)


</details>